specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/restorePoint.json

```diff
@@ -717,10 +717,27 @@
             "lun"
           ],
           "description": "Gets the data disks of the VM captured at the time of the restore point creation."
+        },
+        "diskControllerType": {
+          "$ref": "#/definitions/RestorePointSourceVMDiskControllerType",
+          "readOnly": true,
+          "description": "Gets the disk controller type of the VM captured at the time of the restore point creation."
         }
       },
       "description": "Describes the storage profile."
     },
+    "RestorePointSourceVMDiskControllerType": {
```

Comment:
Consider that 'Gets the disk controller type of the VM captured at the time of the restore point creation' or 'RestorePointSourceVMDiskControllerType' may be overly specializing your type. Maybe you should just call it 'DiskControllerType'

Reply:
Looks like this got fixed already